src/Logary/Targets_Core.fs

```diff
@@ -811,11 +814,11 @@ module File =
     /// This module contains rotation policies that rotate the files after
     /// specific periods.
     module ByAge =
-      let everyMinute = FileAge (Duration.FromMinutes 1L)
-      let every10minutes = FileAge (Duration.FromMinutes 10L)
-      let hourly = FileAge (Duration.FromHours 1L)
-      let daily = FileAge (Duration.FromHours 24L)
-      let weekly = FileAge (Duration.FromHours (7L * 24L))
+      let everyMinute = FileAge (Internals.Date.Duration.fromMinutes 1L)
```

Comment:
Could you explain a bit about why you modelled the API like this? E.g. isn't `Date` implying there's no time: hours, minute, seconds, millis etc?